plugins/sentinel/sentinel/mapping/xdr_alert.py

```diff
@@ -0,0 +1,280 @@
+                    dt_obj = parser.isoparse(timestamp)
+                    # Map specific timestamps to Howler fields
+                    if time_field == "createdDateTime":
+                        howler_hit["event"]["created"] = dt_obj.isoformat()
+                    elif time_field == "firstActivityDateTime":
+                        howler_hit["event"]["start"] = dt_obj.isoformat()
+                    elif time_field == "lastActivityDateTime":
+                        howler_hit["event"]["end"] = dt_obj.isoformat()
+                except Exception:
+                    logger.warning("Invalid timestamp format for %s: %s", time_field, graph_alert[time_field])
+
+
+    def _map_graph_host_link(self, graph_alert: Dict[str, Any], howler_hit: Dict[str, Any]) -> None:
+        """Map Graph host link from Graph alert to Howler hit using the same logic as in xdr_incident.py."""
+        link: Dict[str, str] = {
+            "icon": "https://security.microsoft.com/favicon.ico",
+            "title": "Open in Microsoft XDR portal",
+            "href": graph_alert.get("alertWebUrl", "")
+        }
+        if graph_alert.get("alertWebUrl"):
+            howler_hit["howler"]["links"] = howler_hit["howler"].get("links", [])
+            howler_hit["howler"]["links"].append(link)
+
+
+    def _populate_event_provider(self, howler_hit: Dict[str, Any]) -> None:
+        """Ensure event.provider is populated."""
+        if "provider" not in howler_hit["event"] or not howler_hit["event"]["provider"]:
+            howler_hit["event"]["provider"] = "MSGraphAlertCollector"
+
+    def _populate_comments(
+        self, graph_alert: Dict[str, Any], howler_hit: Dict[str, Any]
+    ) -> None:
+        """Populate comments in the Howler hit."""
+        if "comments" in graph_alert and isinstance(graph_alert["comments"], list):
+            comments = []
+
+            for comment in graph_alert.get("comments", []):
+                values = [
+                    comment.get("comment", comment.get("Comment", None)),
+                    comment.get("createdBy", comment.get("createdByDisplayName", None)),
+                    comment.get("createdTime", comment.get("createdDateTime", None)),
+                ]
+                if not all(values):
+                    logger.info("Invalid comment format in alert: %s", comment)
+                    continue
+                comments.append(
+                    {"value": values[0], "user": f"{values[1]} (Imported from source platform)", "timestamp": values[2]}
+                )
+            if "comment" not in howler_hit["howler"]:
+                howler_hit["howler"]["comment"] = []
+            howler_hit["howler"]["comment"].extend(comments)
+
+    def _map_evidence(
+        self, graph_alert: Dict[str, Any], howler_hit: Dict[str, Any]
+    ) -> None:
+        """Map evidence from Graph alert to Howler hit."""
+        if "evidence" not in howler_hit or not isinstance(howler_hit["evidence"], list):
```

Comment:
We may want to explicitly crash on mapping if the evidence plugin isn't enabled.

```suggestion
        if "evidence" not in config.core.plugins:
            raise NonRecoverableError("Sentinel requires the evidence plugin to be enabled")

        if "evidence" not in howler_hit or not isinstance(howler_hit["evidence"], list):
```